src/components/Markdown/Tiles.tsx

```diff
@@ -0,0 +1,36 @@
+import Link from 'src/components/Link';
+import Icon from '@ably/ui/core/Icon';
+import { IconName } from '@ably/ui/core/Icon/types';
+
+export type TileProps = {
```

Comment:
To keep things cleaner behind the scenes, we don't need to export things unless we intend to use them outside of the file - since only `Tiles` is used, we can just export that.